### PR TITLE
Serializability checker ignores types serialized by Akka internal serializers

### PR DESCRIPTION
**Pull request by MarconZet**

### CODE REVIEW

File: serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala [hunk 1]
```diff
@@ -20,6 +20,29 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val akkaSerializabilityTraits = Seq(
```
Comment (PawelLipski) [inline serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala:23]: Hmmm is the val name adequate? 🤔

File: serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala [hunk 2]
```diff
@@ -40,8 +63,6 @@ class SerializabilityCheckerCompilerPluginComponent(
         ("akka.actor.typed.ActorRef.tell", Seq(ClassType.Message)),
         ("akka.actor.typed.RecipientRef.tell", Seq(ClassType.Message)))
 
-      private val ignoredTypePrefixes = List("akka.", "scala.Any", "scala.Nothing")
```
Comment (PawelLipski) [inline serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala:43]: Actually... how is the `akka.` prefix handled now then?

Comment (MarconZet) [reply serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala:43]: I moved this `val` up the file, to line 44

File: serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala (hunk 1 above)
Comment (MarconZet) [reply serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala:23]: `akkaSerializabilityMarkerTraits`?

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**